app/helpers/terms_and_conditions_helper.rb

```diff
@@ -6,28 +6,10 @@ def link_to_platform_terms
                                                                    rel: "noopener")
   end
 
-  def render_terms_and_conditions
-    if platform_terms_required? && distributor_terms_required?
-      render("checkout/all_terms_and_conditions")
-    elsif platform_terms_required?
-      render "checkout/platform_terms_of_service"
-    elsif distributor_terms_required?
-      render "checkout/terms_and_conditions"
-    end
-  end
-
   def any_terms_required?(distributor)
     TermsOfService.required?(distributor)
   end
 
-  def platform_terms_required?
-    TermsOfService.platform_terms_required?
-  end
-
-  def distributor_terms_required?
```

Comment:
I think the `platform_terms_required?` and `distributor_terms_required?` methods are still in use..?